.travis.yml

```diff
@@ -42,7 +42,7 @@ script:
 - ./gradlew check -S --info --console plain
 
 after_success:
-- bash <(curl -s https://codecov.io/bash) -F unittest
+- test $TRAVIS_BRANCH = "master" && bash <(curl -s https://codecov.io/bash) -F unittest
```

Comment:
So we no longer will do CodeCov in Travis PRs?

Reply:
TRAVIS_BRANCH is the target branch

Reply:
Ah